environments/gcp/README.md

```diff
@@ -0,0 +1,99 @@
+# Flyte on GCP
+
+Prerequisites:
+
+- [terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli#install-terraform)
+
+Procedure:
+
+1. Create a project on GCP
+2. Run ``gcloud auth application-default login``
+3. Confirm the ID of your project:
+    ```bash
+    gcloud projects list
+    ```
+
+3 . Set the default project
+
+```bash
+gcloud config set project <YOUR_PROJECT_ID>
+```
+
+4. Set the default region
+    - ``gcloud config set compute/region <your-GCP-region> ``
+    - [Learn more](https://cloud.google.com/compute/docs/gcloud-compute)
+5.  Enable necessary APIs:
+    - ``gcloud services enable servicenetworking.googleapis.com container.googleapis.com compute.googleapis.com``
```

Comment:
Suggestion:
`container.googleapis.com` and `compute.googleapis.com` could be activated via `environments/gcp/services.tf` instead of this explicit `gcloud` command. If there is a chicken egg problem, it could be resolved with a `depends_on` in the respective resource.